packages/wdio-sauce-service/tests/service.test.ts

```diff
@@ -60,13 +63,13 @@ jest.mock('../src/utils', () => {
 })
 
 test('before should call isUnifiedPlatform', () => {
-    const service = new SauceService()
+    const service: any = new SauceService()
     service.before()
     expect(isUnifiedPlatform).toBeCalledTimes(1)
 })
 
 test('beforeSuite', () => {
-    const service = new SauceService()
+    const service: any = new SauceService()
```

Comment:
Why do we use `any` here instead of instance of SauceService?

Reply:
any comment?

Reply:
Good catch, I probably accidentally set that one here, I just changed it.